org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtBuildReporter.scala

```diff
@@ -88,7 +109,7 @@ private[zinc] class SbtBuildReporter(project: IScalaProject) extends xsbti.Repor
       val markerPos = MarkerFactory.RegionPosition(offset, identifierLength(pos.lineContent, pos.pointer), line)
       BuildProblemMarker.create(resource, severity, msg, markerPos)
     } else
-      logger.error(s"suppressed error in Java file ${resource.getFullPath}:$line: $msg")
+      logger.warn(s"suppressed error in Java file ${resource.getFullPath}:$line: $msg")
```

Comment:
shouldn't it just log to `info` now? Afterwards javac reports the errors now as far as I understand.